docs/multi-module-testing.md

```diff
@@ -103,6 +103,15 @@ You may want to exclude a library module from testing when using Fulladle. You c
       enabled.set(false)
     }
     ```
+
+
+### Overriding root-level config
+Flank offers two different ways to specify APKs for testing, one is the *root-level* configuration (under the `gcloud` namespace) and the other is as an `additional-app-test-apks` (under the `flank` namespace).
+
+Fulladle does not provide the ability to control which module ends up as the root-level module or as an additional module. Either one of app modules or library modules can become a root-level module. If a library module ends up as a root-level module, it needs to specify a `debugApk` in its `fladle` or `fulladleModuleConfig` block.
```

Comment:
You cannot control which module ends up as the root level module but at the same time you need to tell whichever one ends up as the root to have a `debugApk`? How can you do that if you do not know where to add it?

Reply:
Since we can't predict which module becomes root, we have to add `debugApk` to all modules. If you find that a module isn't picked as root, you can remove its `debugApk` field.

Reply:
We should probably document this. But let's do that in a follow up. I will merge this for now.